tests/unit/utils/test_file_readers.py

```
@@ -579,3 +579,40 @@ def test_read_file_dxf(self, tmp_path):
+@pytest.mark.ci
+class TestReadRtfFileCi:
+    """CI-marked tests for read_rtf_file in utils/readers/documents.py."""
+
+    def test_rtf_unavailable_raises_import_error(self, tmp_path: Path) -> None:
+        from utils.readers.documents import read_rtf_file
+
+        rtf_path = tmp_path / "test.rtf"
+        rtf_path.write_text("{\\rtf1 hello}", encoding="latin-1")
+
+        with patch("utils.readers.documents.STRIPRTF_AVAILABLE", False):
+            with pytest.raises(ImportError, match="striprtf is required"):
```

Ruff (0.15.10)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)